pages/sign-up.js

```diff
@@ -17,10 +17,26 @@ const SignUp = () => {
       confirmPassword: yup
         .string()
         .required('Confirm password is required')
-        .oneOf([yup.ref('password')], "Password's not match")
+        .oneOf([yup.ref('password')], "Password's not match"),
+      acceptTerms: yup.boolean().oneOf([true], 'Must Accept Terms and Conditions'),
+      agreeContact: yup.boolean(),
+      acceptMulti: yup
+        .array()
+        .min(1, 'Must Accept Terms and Conditions')
+        .required('At least one checkbox is required')
+        .nullable()
     })
     .required();
 
+  const options = [
+    {
+      name: 'acceptMulti',
```

Comment:
Since the name will be the same for all of the options, it doesn't make sense to include them in the options.